infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/LogReplicationConfig.java

```diff
@@ -37,9 +37,9 @@ public class LogReplicationConfig {
     // Default value for the max number of entries applied in a single transaction on Sink during snapshot sync
     public static final int DEFAULT_MAX_SNAPSHOT_ENTRIES_APPLIED = 50;
 
-    // Max message size supported by protocol buffers is 64MB. Log Replication uses this limit as the default message
+    // Max message size supported by protocol buffers is 25MB. Log Replication uses this limit as the default message
```

Comment:
This is incorrect.  Max message size supported by protocol buffers is not 25 MB.   This value is the default Corfu transaction size.  Can you please update the comment?